src/RMGConvertLH5.cc

```diff
@@ -299,20 +299,28 @@ bool RMGConvertLH5::ConvertToLH5Internal() {
   auto ntuples_group = hfile.openGroup(ntuple_group_name);
   auto ntuples = GetChildren(ntuples_group);
   bool ntuple_success = true;
+  std::cout << std::endl;
   for (auto& ntuple : ntuples) {
+    if (ntuple.size() == 0) LH5Log(RMGLog::fatal, "empty ntuple name, how is this possible?");
```

Comment:
```suggestion
    if (ntuple.empty()) LH5Log(RMGLog::fatal, "empty ntuple name, how is this possible?");
```

otherwise `clang-tidy` will complain